grunt/tasks/jasmine.js

```diff
@@ -21,7 +21,9 @@ module.exports = {
     return {
 
       'src': {
-        src: [], // actual src files are require'd in the *.spec.js files
+        src: [
+          'dist/cartodb.uncompressed.js',
+        ],
```

Comment:
Liked I mentioned earlier offline I'd suggest to extract those particular test-cases, torque-specs bundle loads both cartodb.js and torque bundles for example and would probably be a good place. 
That that the change I proposed in #814 will be covered by these tests too.